src/main/java/uk/gov/di/ipv/cri/common/library/domain/SessionRequest.java

```diff
@@ -22,6 +22,11 @@ public class SessionRequest {
     private SharedClaims sharedClaims;
     private String persistentSessionId;
     private String clientSessionId;
+    private int attemptCount = 0;
+    // TODO: Review the following variables and weather they are needed for passport functionality
+    private String latestDcsResponseResourceId;
+    private String userId;
```

Comment:
Is it possible to prefix the userId with anything meaningful to make it's name a little more specific/descriptive?